### PR TITLE
fossbytes

### LINKED ISSUE #1255: [Add] Fossbytes
### Site URL

https://fossbytes.com/

### Description of the above provided site

Adding Fossbytes blogs which contains the diversified info related tech

### Category

Resources

### Sub Category

blogs

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Shreya111111**
## Fixes Issue
fixes #1255

## Changes proposed
Added fossbytes platform under resource section in blogs.

## Screenshots

Screenshot 2023-07-09 180914

## Note to reviewers
Kindly review it under gssoc'23

### CODE REVIEW

**Comment by Shreya111111** [comment]
@CBID2 and @k-deepak04 kindly review issue #1255 and pr #1268 under gssoc'23

**Review by k-deepak04 — DISMISSED** [review]

**Comment by Shreya111111** [comment]
@rupali-codes kindly review and merge the PR

File: database/resources/blogs.json [hunk 1]
```diff
@@ -145,5 +145,12 @@
     "url": "https://app.daily.dev/",
     "category": "resources",
     "subcategory": "blogs"
+  },
+  {
+    "name": "FossBytes",
+    "description": "Fossbytes is a technology news and information platform that focuses on delivering the latest insights on various aspects of the tech world.",
```
Comment (CBID2) [inline database/resources/blogs.json:151]: [Proofreading]
```suggestion
    "description": "Fossbytes is a platform that focuses on delivering the latest insights on various aspects of the tech world.",
```

Comment (Shreya111111) [reply database/resources/blogs.json:151]: ok

**Comment by Shreya111111** [comment]
@k-deepak04 I have made some changes said by @CBID2 kindly re-review it

**Comment by Shreya111111** [comment]
@k-deepak04 can plz review this again as i have made some changes

**Review by k-deepak04 — APPROVED** [review]

**Comment by k-deepak04** [comment]
@Shreya111111 done

**Comment by Shreya111111** [comment]
@rupali-codes can u kindly review and merge this

**Comment by CBID2** [comment]
There's some conflicts that need to fix

**Comment by Shreya111111** [comment]
@CBID2 I think that was another PR . In this PR,I  changed everything that you mentioned

**Comment by Shreya111111** [comment]
Done necessary additions and deletions
Screenshot 2023-07-11 231206

**Comment by CBID2** [comment]
> Done necessary additions and deletions
> 
> Screenshot 2023-07-11 231206
> 
> 
@Shreya111111 
proof

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Shreya111111! :) Your branch still has merge conflicts.

**Pull request closed without merging**